lib/broadway_cloud_pub_sub/google_api_client.ex

```diff
@@ -204,10 +204,37 @@ defmodule BroadwayCloudPubSub.GoogleApiClient do
 
   defp validate_scope(opts) do
     with {:ok, scope} <- validate(opts, :scope, @default_scope) do
+      ensure_goth_loaded!()
       {:ok, {Goth.Token, :for_scope, [scope]}}
     end
   end
 
+  defp ensure_goth_loaded() do
+    unless Code.ensure_loaded?(Goth.Token) do
+      Logger.error("""
```

Comment:
logging an error instead of raising because otherwise it looks like this:

<img width="2402" alt="image" src="https://user-images.githubusercontent.com/76071/63277756-34efc800-c2a6-11e9-9a10-00d78bacf912.png">